flytekit/models/sagemaker/parameter_ranges.py

```diff
@@ -152,7 +152,8 @@ def to_flyte_idl(self) -> _idl_parameter_ranges.CategoricalParameterRange:
 
     @classmethod
     def from_flyte_idl(cls, pb2_object: _idl_parameter_ranges.CategoricalParameterRange):
-        return cls(values=pb2_object.values)
+        #    return cls(values=pb2_object.values)
```

Comment:
Without doing that, this following test would fail
```
def test_categorical_parameter_range():
    case = unittest.TestCase()
    pr = parameter_ranges.CategoricalParameterRange(values=["abc", "cat"])
    pr2 = parameter_ranges.CategoricalParameterRange.from_flyte_idl(pr.to_flyte_idl())
    case.assertEqual(type(pr2.values), list)
```
with error message
```
  File "/Users/changhonghsu/src/flytekit/tests/flytekit/unit/models/sagemaker/test_parameter_ranges.py", line 40, in test_categorical_parameter_range
    case.assertEqual(type(pr2.values), list)
  File "/Applications/PyCharm.app/Contents/plugins/python/helpers/pycharm/teamcity/diff_tools.py", line 38, in _patched_equals
    raise error
teamcity.diff_tools.EqualsAssertionError:  :: <class 'google.protobuf.pyext._message.RepeatedScalarContainer'> != <class 'list'>
----------------------------------------------------------------------
Ran 1 test in 0.034s
FAILED (failures=1)
```